src/table/styles.scss

```diff
@@ -152,6 +152,9 @@ filter search icon.
   margin-top: -15px;
   bottom: 0;
   width: 100%;
+  &-with-sticky-columns {
+    z-index: 801; // Same as sticky columns
```

Comment:
Can we update this unconditionally? I can imagine this breaks something else like the scrollbar can be over split-panel or something, but it will break it anyways if the sticky columns is on.

Reply:
That's a good point. I was thinking of the selection cells on non sticky columns, but we would also have that problem in sticky columns, so I would just be minimising the blast radius here.
I'll update it.